components/carousel/carousel.css

```diff
@@ -0,0 +1,83 @@
+.ui-carousel {
+  overflow: hidden;
+  position: relative;
+}
+
+.ui-carousel-item {
```

Comment:
Although I understand and also code like that, @reaktivo, from other source codes I've read, this isn't really being enforced. So I'd like to know if we're going to do so. Not only that, the helper that generates the mods class names doesn't apply mods as '--'.

Any thoughts on that, @mAiNiNfEcTiOn @asci ?